dist/image-sequencer.js

```diff
@@ -25923,23 +25912,15 @@ function binarize(data, width, height, returnInverted) {
                 }
             }
             var threshold = sum / 25;
-            for (var xRegion = 0; xRegion < REGION_SIZE; xRegion++) {
-                for (var yRegion = 0; yRegion < REGION_SIZE; yRegion++) {
-                    var x = hortizontalRegion * REGION_SIZE + xRegion;
-                    var y = verticalRegion * REGION_SIZE + yRegion;
-                    var lum = greyscalePixels.get(x, y);
-                    binarized.set(x, y, lum <= threshold);
-                    if (returnInverted) {
-                        inverted.set(x, y, !(lum <= threshold));
-                    }
+            for (var x = 0; x < REGION_SIZE; x++) {
```

Comment:
Just worried about this part, can you try rebuilding dist files to be sure we're not reverting something important?